commandline/WSL/install_guide.md

```diff
@@ -1,84 +1,93 @@
+    Here are links directly to the store installers:
+    * [Ubuntu](https://www.microsoft.com/store/p/ubuntu/9nblggh4msv6)
+    * [OpenSUSE](https://www.microsoft.com/store/apps/9njvjts82tjx)
+    * [SLES](https://www.microsoft.com/store/apps/9p32mwbh6cns)
 
-### Enable the Windows Subsystem for Linux feature (command-line)
+    ![](media/UbuntuStore.png)
 
-Open a PowerShell prompt as administrator and run:
+2. Select "Get"
 
-```PowerShell
-Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
-```
+    >  **Troubleshooting:  Installation failed with error 0x8007007e**  
+    >  This error occurs when your system doesn't support Linux from the store.  Make sure that:
+    > * You're running Windows build 16215 or later. [Check your build](install_guide.md#prerequisites).
+    > * The Windows Subsystem for Linux optional component is enabled. [Instructions here](install_guide.md#Install-the-Windows-Subsystem-for-Linux).
+    
+3. Once the download has completed, select "Launch".  
+    This will open a console window.  Wait for installation to complete then you will be prompted to create your UNIX user account.
+    ![](media/UbuntuInstall.png)
+    
+5. Create your UNIX username and password.  This user account can be different from, and has no relationship to, your Windows username and password. [Read more](https://msdn.microsoft.com/en-us/commandline/wsl/user_support).
 
-### After enabling Windows Subsystem for Linux
-**Restart your computer when prompted**
+You're done!  Now you can use your Linux environment.
 
-> It is important that you **DO restart when prompted** as some of the infrastructure which Bash on Windows requires can only be loaded during Windows' boot-up sequence.
+## Install using lxrun (Anniversary Update and Creators Update)
 
-### Run Bash on Windows
-1. Open a command prompt
-1. Run `bash` 
-  
-  ![](media/bashShellInstall.png)
-  
-After you have accepted the License, the Ubuntu user-mode image will be downloaded and a "Bash on Ubuntu on Windows" shortcut will be added to your start menu.
+1. Turn on Developer Mode  
 
-To launch Bash on Windows, either run `bash` at a cmd/PowerShell command-prompt, or use the start menu shortcut.
+    Open **Settings** -> **Update and Security** -> **For developers**
 
-After installation your Linux distribution will be located at: `%localappdata%\lxss\`
-This directory is marked as a hidden system folder for a very good reason: 
+    Select the Developer Mode radio button  
+    ![](media/updateAndSecurity.png)
 
-> **Avoid creating and/or modifying files in this location using Windows tools 
-> and apps!** If you do, it is likely that your Linux files will be corrupted and data loss may occur. 
-> Please read this [blog post](https://blogs.msdn.microsoft.com/commandline/2016/11/17/do-not-change-linux-files-using-windows-apps-and-tools/) 
-> for more information.
+    Restart your Computer.

```

Comment:
Good catch -- forgot to scrub it when I pulled the optional component enabling steps out